docs/spec.md

```diff
@@ -538,6 +561,20 @@ All integer types explicitly specified (uint32, uint64, etc.) in this section ar
 | _encoding length_ | char[]  | encoding, same encoding that was used for the request |
 | remaining bytes   | uint8[] | response payload                                      |
 
+### Asset
+
+- Response to a [fetch asset](#fetch-asset) request if the asset was found
+- Only supported if the server previously declared the `assets` [capability](#server-info).
+
+| Bytes               | Type    | Description                                                                                            |
+| ------------------- | ------- | ------------------------------------------------------------------------------------------------------ |
+| 1                   | opcode  | 0x04                                                                                                   |
+| 4                   | uint32  | request id, as given in the corresponding [request](#fetch-asset)                                      |
+| 8                   | uint64  | Last modified time of the asset (nanoseconds)                                                          |
```

Comment:
Not sure if the last modified time is very relevant for the client. Maybe we can omit it

Reply:
can't really come up with a use case, will remove it